lyrics-app/pages/index.js

```diff
@@ -0,0 +1,139 @@
+import axios from "axios";
+import { useState } from "react";
+
+export default function Home() {
+  const [title, setTitle] = useState(null);
```

Comment:
You can add a title by default so that anyone can just search for it by clicking on the button.